gamemodes/terrortown/gamemode/client/cl_targetid.lua

```diff
@@ -396,7 +397,7 @@ function HUDDrawTargetIDWeapons(data, params)
 	local client = LocalPlayer()
 
 	if not IsValid(client) or not client:IsTerror() or not client:Alive()
-	or data.distance > 100 or not data.ent:IsWeapon() then
+	or data.distance > 1000 or not data.ent:IsWeapon() then
```

Comment:
is this a testing change? because the distance "100" is our default distance

Reply:
@TimGoll

Reply:
already resolved. Maybe your mobile website did not refresh it